libraries/AP_HAL_ChibiOS/Storage.h

```diff
@@ -29,7 +29,13 @@
 
 // when using flash storage we use a small line size to make storage
 // compact and minimise the number of erase cycles needed
+#ifdef STORAGE_FLASH_PAGE
+#define CH_STORAGE_LINE_SHIFT 3
+#elif defined(USE_POSIX)
```

Comment:
needs to be defined(USE_POSIX) && !defined(HAL_WITH_RAMTRON)

Reply:
Fixed